include/common.sh

```diff
@@ -244,6 +255,26 @@ LIBPATH=\${ZOPEN_ROOTFS}/usr/local/lib:\${ZOPEN_ROOTFS}/usr/lib:\$(sanitizeEnvVa
 export LIBPATH=\$(deleteDuplicateEntries "\${LIBPATH}" ":")
 MANPATH=\${ZOPEN_ROOTFS}/usr/local/share/man:\${ZOPEN_ROOTFS}/usr/local/share/man/\%L:\${ZOPEN_ROOTFS}/usr/share/man:\${ZOPEN_ROOTFS}/usr/share/man/\%L:\$(sanitizeEnvVar "\${MANPATH}" ":" "^\${ZOPEN_PKGINSTALL}/.*\$")
 export MANPATH=\$(deleteDuplicateEntries "\${MANPATH}" ":")
+
+if \${knv}; then
+  env | sort > /tmp/zopen-config-env-modded.\$\$
+  diffout=\$(diff /tmp/zopen-config-env-orig.$\\$ /tmp/zopen-config-env-modded.\$\$ | grep -E '^[>]' | cut -c3- )
```

Comment:
was this tested with both /bin/diff and z/OS Open Tools diff? Same question for grep. 
Wondering if we should just hardcode /bin/

Reply:
we should just hardcode /bin/diff to ensure consistency.